Gemfile

```diff
@@ -3,3 +3,5 @@
 source "https://rubygems.org/"
 
 gemspec
+
+gem 'csv', github: 'ruby/csv'
```

Comment:
I've released csv gem. Could you remove this and updated required csv version to 3.2.4?

Reply:
OK 👍 Thanks!

Reply:
I appreciate if you could see it.